okta/okta-sdk-appauth-android
# Adding state param to config

Implementation for #32

Steps to test:
1. Put a breakpoint at OktaAppAuth:503 or at that line add a log `Log.d(TAG, "Browser will load:" + authRequestBuilder.build().toUri());`
1. Within okta_app_auth_config.json add `"state": "&someParam=2"`
1. Run the demo app
1. login with a valid user
    * Confirm that the URI contains `state=%someParam%3D2`
